bsoid_app/data_preprocess.py

```diff
@@ -10,216 +10,175 @@
 from bsoid_app.bsoid_utilities.likelihoodprocessing import *
 from bsoid_app.bsoid_utilities.load_json import *
 
+software_choice = os.environ.get('software_choice', '')
+ftype = os.environ.get('ftype', '')
+framerate = os.environ.get('framerate', '')
+value = float(os.environ.get('value', ''))
+data_directories = os.environ.get('data_directories', '')
+data_directories = data_directories.split(',')
+root_path = os.environ.get('root_path', '')
+working_dir = os.environ.get('working_dir', '')
+prefix = os.environ.get('prefix', '')
+pose_list = os.environ.get('pose_list', '')
```

Comment:
I guess i don't understand what os.environ.get is doing here/throughout, is this a way of specifying arguments?

Reply:
I set environmental variables, and this retrieving them. The organization has changed slightly based on Mert's advice but still using environmental variable because I don't have a better solution...